### DISCUSSION TITLE
Debloat issue report on Xiaomi 13T phone

### DISCUSSION THREAD

**Discussion by tim77** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#443] [Q&A] [answered]
One of the Recommended packages to remove result into deleting some handy stuff on Xiaomi 13T phone. Fingerprint verification regression on this device after de-bloat. In particular if this feature enabled user have somewhat always on screen feature with circle⭕ on the screen to help find where user should to press to unblock device. It appears when user take phone in the hand. But after de-bloating this feature doesn't work. But unblocking via fingerprint scanner still possible via double tap firs, but not convenient anymore.

Unfortunately i don't found solution yet. But hope we could find or maybe someone already figured out what package is responsible for this feature. Please help if you know.

**Comment by Blitz517** [comment]
you can find it only if uad-ng detects device

**Comment by kaillantis** [comment, marked as answer]
I have a Redmi 13 Pro 5G and found the same problem. It was fixed by reenabling com.miui.aod
I would recommend deleting this package from the recommended list.

**Comment by AnonymousWP** [comment]
Fixed in https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/478.
